lib/texts.js

```diff
@@ -251,6 +251,7 @@ ${leaves}`,
 	cliLogLevel: () => 'Logger verbosity level',
 	cliDisallowCrashReports: () => 'Disallow Suitest to submit automatic crash reports when unexpected exceptions occur',
 	cliContinueOnFatalError: () => 'Do not force process exit when suitest chain fails with fatal error',
+	cliTimestamp: () => 'Logger timestamp format. Use "none" to disable',
```

Comment:
Some example or at least default value would be nice. Otherwise user wouldn't have a clue on how to define format.

Reply:
Not sure if `yargs` displays default value in some form. Perhaps it's enough to put default value there.

Reply:
Yargs supports default option, but unfortunately with current implementation we override config with cli args, so if user has something in RC file it will be overridden by default option if the same cli arg is not present. Similar to logLevel, we don't specify default option for it.

Reply:
OK, then default value should be in description